wordle-web/src/components/AvailableWordsButton.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <v-menu :close-on-content-click="false" v-model:model-value="showMenu">
```

Comment:
I think there's supposed to be a hover element in this component, to make sure/highlight the word you're hovering over?